test/docs.jl

```diff
@@ -69,6 +69,12 @@ $$latex literal$$
 """
 function break_me_docs end
 
+
+# `hasdoc` returns `true` on a name with a docstring.
+@test Docs.hasdoc(Base, :map)
+# `hasdoc` returns `false` on a name without a docstring.
+@test !Docs.hasdoc(Base, :_this_name_doesnt_exist_)
```

Comment:
Would be good to add a test for a name that does exist but isn't documented, e.g. 
```suggestion
@test !Docs.hasdoc(Base, :_this_name_doesnt_exist_)
@test isdefined(Base, :_typed_vcat) && !Docs.hasdoc(Base, :_typed_vcat)
```